ui/components/config/provider-form.tsx

```
@@ -806,6 +832,15 @@ export default function ProviderForm({ provider, onSave, onCancel, existingProvi
                                     className="transition-all duration-200 ease-in-out"
                                   />
                                 </div>
+                                <div>
+                                  <label className="text-sm font-medium">Region (Required)</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)